command/server/server.go

```diff
@@ -171,6 +181,26 @@ func (s *Server) setupGRPCServer(addr string) error {
 	return nil
 }
 
+func (s *Server) StartPrometheus(address string, enabled bool) error {
+	// Hook go-metrics into expvar on any /debug/metrics request, load all vars
+	// from the registry into expvar, and execute regular expvar handler.
+	if enabled {
+		exp.Exp(metrics.DefaultRegistry)
```

Comment:
Instead of calling this function from the metrics package and since the function is so little (3 lines). Just copy the content of the function here.